source/guide/3.2/dsl-spec-policy-types.md

```diff
@@ -44,21 +44,61 @@ Example:
 {%highlight yaml%}
 policy_types:
 
+  my_host_failure_policy:
```

Comment:
The example is incorrect. properties in policy types are in schema structure, i.e:
``` yaml
properties:
       policy_operates_on_group: 
           description: The policy will maintain its state for each node instance individually.
           default: false
```